docs/parameters.rst

```diff
@@ -8,9 +8,31 @@ This class is an attempt to group them all inside one single structure.
 For now, :class:`Parameters` have default value, but the can be read back / from json. Other serialization method will
 come soon.
 
+Example
+--------
+.. code-block:: python
```

Comment:
Can you add a sentence like "If you want to change the parameters it is better to do it at the creation of the environment. This can be done with: